content/ui-patterns/empty-states.mdx

```diff
@@ -0,0 +1,58 @@
+
+### Graphic
+The graphic can bring delight, preview an interface element, or represent the goal of the feature. Graphics should be placed intentionally and with thought about the intention of the content. Graphics also differ in meaning and appeal to the user, which is why the blankslate component has multiple variations. These variations are outlined later on this page.
+
+### Primary Text
+Use primary text to explain the purpose of the empty state, help users feel comfortable to engage with the content, or begin a feature flow. Primary text should sound welcoming, human, and convey the intention of the feature.
+
+### Secondary text
+This optional text is used to inform the user about the feature in more detail. Secondary text should be brief and non-redundant if possible. From the text, users should  be able to understand the general purpose of the feature and how it may help them accomplish a goal.
+
+### Primary action
+Blankslates can and are encouraged to use one primary action. This button should lead to a feature or component creation flow. Button copy should be keep brief and descriptive. If a button requires further specification, consider adding an Octicon.
+
+### Secondary action
+Secondary actions are optional and are represented by a text link located below the primary action button. A secondary action is used to direct a user to additional content about the feature. This might look like "[Learn more about X](#)" or "[Check out the guide on X](#)" or simply "[Learn more](#)".
+
+### Border
+The border is invisible by default, but can be added to help define the structure of the blankslate component when needed. This can be particularly helpful in page layouts where the blankslate is not the only content on the screen.
+
+## Variations
+Empty states have multiple variations that can be used in different contexts.
+
+### Pictograms
+[Pictograms](https://ghicons.github.com/) present a great preview of what should be in place of the empty state. Blankslates should default to using a pictogram for graphic elements.
+
+![pictogram blankslate](https://user-images.githubusercontent.com/6846673/62806619-39e39900-baa8-11e9-9de0-5b2f511d63da.png)
+
+### Illustrations
+Illustrations should be used **sparingly** in blankslates - only in cases where it is necessary to evoke emotion from the user. For example, illustrations in a new user experience can help introduce octocats to the user and bring a more personalized experience.
+
+![illustration blankslate](https://user-images.githubusercontent.com/6846673/62806617-394b0280-baa8-11e9-9abc-ccbb76a9f63f.png)
+
+### Compact
+Compact states are a great way to engage and educate a user within Organizations, Settings, and other nested navigation pages. They should replace content that is left-aligned. While the border is optional for all empty states, the Compact variation requires it to better support page structure.
```

Comment:
I'm not sure if it's necessary to list out specific areas in the product for compact blankslates. I think it provides more understanding and flexibility if we write the guideline to show that compact blankslates  are use when there are more UI elements and the area is much more condensed.

Reply:
Also I feel a little unclear on 
> They should replace content that is left-aligned.
The way this is written it feels as if all of the left-aligned content will turn into a blankslate.

Reply:
Ok, I can reword this.